### PR TITLE
Fixes [Bug]: Prod/beta flavor messages may be nonsense for users #4850

### PR DESCRIPTION
**Pull request by Rishavgupta12345**
Fixes [Bug]: Prod/beta flavor messages may be nonsense for users #4850

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -689,7 +689,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="no_location_found_message">How about adding the place where this image was taken?\nLocation data helps Wiki editors find your picture, making it much more useful.\nThank you!</string>
   <string name="add_location">Add location</string>
   <string name="feedback_sharing_data_alert">Please remove from this email any information that you are not comfortable sharing publicly. Also, please be aware that your email address with which you are posting, and the associated name and profile picture, will be visible publicly.</string>
-  <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor.</string>
-  <string name="leaderboard_unavailable_beta">The leaderboard is only available in the prod flavor.</string>
+  <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor,Please check the developer documentation.</string>
```
Comment (4D17Y4) [inline app/src/main/res/values/strings.xml:692]: space after comma and small p.

**Review by 4D17Y4 — CHANGES_REQUESTED** [review]
Small change

**Review by Rishavgupta12345 — COMMENTED** [review]
done the changes @4D17Y4

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Rishavgupta12345) [reply app/src/main/res/values/strings.xml]: done

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -689,7 +689,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="no_location_found_message">How about adding the place where this image was taken?\nLocation data helps Wiki editors find your picture, making it much more useful.\nThank you!</string>
   <string name="add_location">Add location</string>
   <string name="feedback_sharing_data_alert">Please remove from this email any information that you are not comfortable sharing publicly. Also, please be aware that your email address with which you are posting, and the associated name and profile picture, will be visible publicly.</string>
-  <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor.</string>
-  <string name="leaderboard_unavailable_beta">The leaderboard is only available in the prod flavor.</string>
+  <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor, Please check the developer documentation.</string>
```
Comment (misaochan) [inline app/src/main/res/values/strings.xml:692]: "Please" needs to be "please", no capital "P".

**Review by Rishavgupta12345 — COMMENTED** [review]
done the changes @misaochan @4D17Y4

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Rishavgupta12345) [reply app/src/main/res/values/strings.xml]: done

**Review by 4D17Y4 — APPROVED** [review]

**Comment by 4D17Y4** [comment]
Thanks @Rishavgupta12345

**Pull request merged**